cordova-lib/package.json

```diff
@@ -40,7 +40,7 @@
         "underscore": "1.7.0",
         "unorm": "1.3.3",
         "valid-identifier": "0.0.1",
-        "xcode": "0.8.0"
+        "xcode": "0.8.6"
```

Comment:
Will confirm this is indeed the released version as soon as alunny/node-xcode#88 gets merged as a release is pushed out.